docs/Bbb.md

```diff
@@ -0,0 +1,97 @@
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg -o /etc/apt/keyrings/docker.asc
+chmod a+r /etc/apt/keyrings/docker.asc
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/ubuntu \
+  $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | \
+  tee /etc/apt/sources.list.d/docker.list > /dev/null
+apt-get update
+apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+```
+as per the docker doc recommands, you can try and assure Docker is well installed by running :
+`docker run hello-world`
+
+Then to install latest BBB software (version 2.7), also as per the official doc recommands (read and change two variables before running) :
+Options used :
+`-e` is the email your will provide for being contacted regarding Letsencrypt SSL certificates, you can drop that option if generated your own certificates yourselves
+`-s` is the name your BBB server will be configured with
+`-v` is the BBB software version you're querying
+```
+wget -qO- https://raw.githubusercontent.com/bigbluebutton/bbb-install/v2.7.x-release/bbb-install.sh | bash -s -- -v focal-270 -s YOUR_SERVER_HOSTNAME -e email_for_letsencrypt_certbot@your_domain.org
+echo 'disableRecordingDefault=true' >>  /etc/bigbluebutton/bbb-web.properties
+echo 'breakoutRoomsRecord=false' >> /etc/bigbluebutton/bbb-web.properties
+/usr/bin/bbb-conf --restart
+```
+The two 'echo' commands are for disabling recording.
+
+Then, to ensure your BBB server is up and running, you can run :
+`/usr/bin/bbb-conf --check`, last line you should see in the output is : `# Potential problems described below`
+If this is not the case, please refer to official BBB documentation.
+
+
+### Deploy liquid with the BBB frontend (Greenlight)
+
+As of now, we have to rely on a specific branch from liquid-core to deploy liquid-node with BBB enabled.
+To do that, after having followed the liquid-node installation instructions, please create a versions.ini file at the root of liquid-node, with following content :
+```
+[versions]    
+liquid-core = liquidinvestigations/core:tmp-admin-show-oauth2
+```
+Then run `liquid deploy` again.
+That version will match criteria for the BBB frontend OIDC + it will allow to have a new section "oauth2 provider" with manual access to add new oauth2 app.
+
+Now, to deploy liquid with the BBB frontend, you should get the credentials from your BBB server, to do so, connect to the BBB server and use the bbb-conf tool like this :
+`/usr/bin/bbb-conf --secret`
+that will give you the BBB URL and SECRET to fill in the liquid.ini file, copy-paste them as follows :
+```
+bbb_endpoint = YOUR_BBB_API_URL
+bbb_secret = YOUR_BBB_API_TOKEN
+```
+
+Then, you have to create a connect agent on the liquid admin interface, it's located at :
+`https://YOUR_LIQUID_FQDN/admin/oauth2_provider/application/`
+The info to fill in is as follows :
+```
+client type : 'confidential'
```

Comment:
let's automate the creation of SSO stuff

Reply:
when using 'core_oauth_apps' to automatically create the SSO app, I don't get to the right config, I lack having Algorithm: set as HMAC with SHA-2 256. ( and, skip authorization set as false ).

Could I (or anyone) add a commit into liquid-core ./liquidcore/home/management/commands/createoauth2app.py regarding this issue ? Just so that I can get something like :

```
        if options['noskip'] :
          app.skip_authorization = False
        else:
          app.skip_authorization = True

        if options['algo'] :
          app.algorithm = options['algo']
```
in https://github.com/liquidinvestigations/core/blob/c0b52b240f1305e1ed28acb74644d5a3d6d8b173/liquidcore/home/management/commands/createoauth2app.py#L22

Otherwise, I got into the following code, then error, within django-oauth-toolkit :
https://github.com/jazzband/django-oauth-toolkit/blob/40003dddda5b13f91b841ca267fd730487338c89/oauth2_provider/models.py#L258
aka "This application does not support signed tokens"

Reply:
Done and tested on my part, LGTM, code added on liquid-core https://github.com/liquidinvestigations/core/pull/211